spring-kafka/src/test/java/org/springframework/kafka/annotation/EnableKafkaIntegrationTests.java

```diff
@@ -160,7 +160,7 @@ private void waitListening(String id) throws InterruptedException {
 		public Map<String, Object> consumerConfigs() {
 			Map<String, Object> props = new HashMap<>();
 			props.put("bootstrap.servers", embeddedKafka.getBrokersAsString());
-//			props.put("bootstrap.servers", "localhost:9092");
+			//props.put("bootstrap.servers", "localhost:9092");
```

Comment:
Yeah... Me too.
I guess you fixed that with the `BLOCK_COMMENT_BEGIN`.
Thanks.